src/Brouter/Bit.Brouter.Demo/Client/wwwroot/app.css

```
@@ -134,89 +258,104 @@
-
-    --bb-font-sans: "Segoe UI", "Segoe UI Web (West European)", -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue", Arial, sans-serif;
-    --bb-mono: ui-monospace, SFMono-Regular, "SF Mono", Menlo, Consolas, "Liberation Mono", monospace;
-}
-
+    --bb-content-width: 1320px;
+    --bb-measure: 74ch;
+
+    /* Segoe UI Variable is Fluent 2's typeface; static Segoe UI is the fallback on Windows
+       versions that do not ship it, and the platform stack takes over everywhere else. */
+    --bb-font-sans: "Segoe UI Variable Text", "Segoe UI Variable Static Text", "Segoe UI",
+                    "Segoe UI Web (West European)", -apple-system, BlinkMacSystemFont, Roboto,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)